seacrowd/sea_datasets/filipino_hatespeech_tiktok/filipino_hatespeech_tiktok.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+import datasets
+
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{hernandez2021bert,
+  title={A bert-based hate speech classifier from transcribed online short-form videos},
+  author={Hernandez Urbano Jr, Rommel and Uy Ajero, Jeffrey and Legaspi Angeles, Angelic and Hacar Quintos, Maria Nikki and Regalado Imperial, Joseph Marvin and Llabanes Rodriguez, Ramon},
+  booktitle={2021 5th International Conference on E-Society, E-Education and E-Technology},
+  year={2021}
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["tgl"]
+_DATASETNAME = "filipino_hatespeech_tiktok"
+
+_DESCRIPTION = """\
+The dataset contains annotated hate speech from transcribed Tiktok videos, mostly in Taglish (codemixed Tagalog and Cebuano) collected via an unofficial Tiktok API. Most of the domain and context of the videos are related to politics and general elections in the Philippines. Labeling: 0 - no hate speech, 1 - recognized hate speech
+"""
+
+_HOMEPAGE = "https://github.com/imperialite/filipino-tiktok-hatespeech"
+_LICENSE = Licenses.UNKNOWN.value
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_URLS = {
+    "raw": "https://raw.githubusercontent.com/imperialite/filipino-tiktok-hatespeech/main/data",
+}
+
+
+class FilipinoHateSpeechTikTok(datasets.GeneratorBasedBuilder):
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="filipino_hatespeech_tiktok_source",
+            version=SOURCE_VERSION,
+            description="Filipino TikTok Hatespeech source schema",
+            schema="source",
+            subset_id="filipino_hatespeech_tiktok",
+        ),
+        SEACrowdConfig(
+            name="filipino_hatespeech_tiktok_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="Filipino TikTok Hatespeech Seacrowd schema",
+            schema="seacrowd_text",
+            subset_id="filipino_hatespeech_tiktok",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "filipino_tiktok_hatespeech_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features({"text": datasets.Value("string"), "label": datasets.Value("string")})
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text.features()
```

Comment:
By default, the[ class features here](https://github.com/williamnixon20/seacrowd-datahub/blob/4a8300b136a96494f65dcb1cfdc010f6255f773d/seacrowd/utils/schemas/text.py) are `"Yes"` and `"No"`. We want to be consistent with what's in the dataset, so I suggest passing "0" and "1" here:

```suggestion
            features = schemas.text.features(label_names = ["0", "1"])
```

You can also create a constant, `CLASS_LABELS` and pass that instead.